src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,10 @@
+package duke.command;
+
+import duke.main.Ui;
+import duke.utils.Storage;
+import duke.utils.TaskList;
+
+public interface Command {
+
```

Comment:
Hi, perhaps you may have been tired, but there's a blank space here which may need removing.
Alternatively, I think it may be aesthetically pleasing if we left another blank space after the line of code before we close the Command class.